connector/interface.go

```diff
@@ -31,8 +42,18 @@ type Connector interface {
 
 //go:generate genconfig -o config.go connector Connector
 type ConnectorConfig interface {
+	// ConnectorID returns an end user facing identifier. For example "google".
 	ConnectorID() string
+
+	// ConnectorType returns an implementation specific identifier. For example "oidc".
 	ConnectorType() string
+
+	// Connector is envoked by the dex server and returns a Connector configured
+	// to use the provided arguments. URL namespace is used to register callbacks.
+	// loginFunc is used to associate remote identies with dex session keys.
```

Comment:
loginFunc is used to associate remote identities with dex session keys. The returned Connector must call loginFunc once upon successful identification of a user.